spec/classes/nginx_spec.rb

```diff
@@ -41,21 +41,24 @@
         when 'RedHat'
           context 'using defaults' do
             it { is_expected.to contain_package('nginx') }
+
             it do
               is_expected.to contain_yumrepo('nginx-release').with(
-                'baseurl'  => "https://nginx.org/packages/#{%w[CentOS VirtuozzoLinux].include?(facts[:operatingsystem]) ? 'centos' : 'rhel'}/#{facts[:operatingsystemmajrelease]}/$basearch/",
```

Comment:
Isn't there also a style that accepts it consistent? So either all aligned or all ` => `?

Reply:
This was added in https://github.com/voxpupuli/voxpupuli-test/pull/77